packages/pipeline-editor/src/CustomFormControls/NumberControl.tsx

```diff
@@ -49,8 +49,9 @@ const ErrorMessage = styled.div`
   z-index: 1;
   border-style: solid;
   border-width: 1px;
-  border-color: #be1100;
-  background-color: #5a1d1d;
+  border-color: ${({ theme }) => theme.palette.errorBorder};
+  background-color: ${({ theme }) => theme.palette.background.error};
+  color: ${({ theme }) => theme.palette.primary.contrastText};
```

Comment:
I think these should come from the error style, right?
```js
background-color: ${({ theme }) => theme.palette.error.main};
color: ${({ theme }) => theme.palette.error.contrastText};
```

Reply:
I just reused an existing style, but it makes sense to keep in in the same context